### DISCUSSION TITLE
What data types can be used to repeat a set of attributes within a data object?

### DISCUSSION THREAD

**Discussion by bhagya7977** [pimcore/pimcore#16278] [Q&A / Support]
Fieldcollections
Blocks
Object Bricks
Classification Store
ManyToMany Relation

**Comment by jantudjan99** [comment]
Did you figure out the correct answer maybe?

**Reply by bhagya7977** [reply]
Yeah got the solution !

**Reply by jantudjan99** [reply]
Can you tell me which answers are correct and why? I'm struggling with the same question.

**Reply by bhagya7977** [reply]
FieldCollections and Blocks  are the correct answers and these data types can be able to repeat the  elements under the DataObject

**Reply by jantudjan99** [reply]
Thank you, this is correct indeed! I was doubting between FieldCollections, Blocks and Object Bricks and just FieldCollections and Blocks but I had just one attempt this week.
